pkgs/development/compilers/ispc/default.nix

```diff
@@ -0,0 +1,55 @@
+{stdenv, fetchFromGitHub, which, m4, python, bison, flex, llvmPackages}:
+
+# TODO: patch LLVM so Knights Landing works better (patch included in ispc github)
+
+stdenv.mkDerivation rec {
+  version = "20151128";
+  rev = "d3020580ff18836de2d4cae18901980b551d9d01";
+
+  name = "ispc-${version}";
+
+  src = fetchFromGitHub {
+    owner = "ispc";
+    repo = "ispc";
+    inherit rev;
+    sha256 = "15qi22qvmlx3jrhrf3rwl0y77v66prpan6qb66a55dw3pw2d4jvn";
+  };
+
+  enableParallelBuilding = true;
+
+  doCheck = true;
+
+  buildInputs = with llvmPackages; [
+    which
+    m4
+    python
+    bison
+    flex
+    llvm
+    clang
+  ];
+
+  patchPhase = "sed -i -e 's/\\/bin\\///g' -e 's/-lcurses/-lncurses/g' Makefile";
+
+  installPhase = ''
+  mkdir -p $out/bin
```

Comment:
Maybe indent the bash code with two more spaces. That is between also the convention described in the nixpkgs manual.